### ISSUE TITLE
View number of reads for heatmap

### ISSUE THREAD

**Issue by gibcus** [higlass/higlass#782]
When comparing heatmaps, I would love to see the number of reads that went into the heatmap

**Comment by pkerpedjiev** [comment]
@nvictus Is there a way to extract this from a cooler file?

**Comment by nvictus** [comment]
yeah, there's a `sum` metadata attribute at every resolution (which should be the same for all)

**Comment by gibcus** [comment]
@manzt, I think this is still relevant for v2 if not already implemented.

**Comment by manzt** [comment]
Thanks for pinging to prioritize! To my knowledge it's not planned, but I'll reopen so it doesn't get lost. @pkerpedjiev

**Comment by manzt** [comment]
My understanding this sounds like some new cooler-specific data we need to send to be displayed in the front end for a heatmap track.
